tests/foreman/virtwho/cli/test_esx_sca.py

```diff
@@ -502,3 +505,95 @@ def test_positive_remove_env_option(
+            4. rhsm.log contains the correct account
+            5. After deleting all virt-who config belong to the same org, verify that the rhsm user belong to this org has been deleted
+
+
+
+        :customerscenario: true
+
+        :CaseImportance: Medium
+
+        :BZ: 2063218
+        """
+        # create two virt-who configs in the same organization, get the service account rhsm_username
+        vc_id = []
+        rhsm_username = []
+        for _i in range(2):
+            form_data_cli['name'] = gen_string('alpha')
+            virtwho_config_cli = target_sat.cli.VirtWhoConfig.create(form_data_cli)[
+                'general-information'
+            ]
+            command = get_configure_command(virtwho_config_cli['id'], module_sca_manifest_org.name)
+            deploy_configure_by_command(
+                command, form_data_cli['hypervisor-type'], org=module_sca_manifest_org.label
+            )
+            vc_id.append(virtwho_config_cli['id'])
+            config_file = get_configure_file(virtwho_config_cli['id'])
+            rhsm_username.append(get_configure_option('rhsm_username', config_file))
+
+        # verify the two service accounts belonging to the same org are the same
+        assert rhsm_username[0] == rhsm_username[1]
+
+        # Create a different org virtwho_fake_XXXX and then create virt-who config in that org, get the service account rhsm_username
+        ORG_DATA = {'name': f'virtwho_fake_{gen_string("alpha")}'}
+        org = target_sat.api.Organization(name=ORG_DATA['name']).create()
+        target_sat.api.Location(organization=[org]).create()
+        form_data_cli['organization-id'] = org.id
+        form_data_cli['name'] = gen_string('alpha')
+        virtwho_config_cli = target_sat.cli.VirtWhoConfig.create(form_data_cli)[
+            'general-information'
+        ]
+        command = get_configure_command(virtwho_config_cli['id'], org.name)
+        deploy_configure_by_command(
+            command, form_data_cli['hypervisor-type'], debug=True, org=org.label
+        )
+        config_file = get_configure_file(virtwho_config_cli['id'])
+        rhsm_username.append(get_configure_option('rhsm_username', config_file))
+        vc_id.append(virtwho_config_cli['id'])
+
+        # verify the two service accounts belonging to different orgs are different
+        assert rhsm_username[2] != rhsm_username[0]
+
+        # Verify rhsm.log contains correct orgs to which hypervisor-guest mapping is sent to, and rhsm.log contains the correct user names.
+        runcmd(get_configure_command(vc_id[0], module_sca_manifest_org.name))
+        runcmd(get_configure_command(vc_id[1], module_sca_manifest_org.name))
+        service_account_message = [
+            f"Authenticating with RHSM username {rhsm_username[0]}",
+            f"Authenticating with RHSM username {rhsm_username[2]}",
+            f"Host-to-guest mapping being sent to '{module_sca_manifest_org.name}'",
```

Comment:
Please use single quotes wherever possible, to ensure consistency :)

Reply:
@Gauravtalreja1 Thank you very much for your suggestion, this used double quotes , it is because there is single quotes in the string, so used double quotes.

Reply:
@yanpliu yes, but we can switch it with single quotes with double quotes inside, but I wouldn't block this PR merge for it